homeassistant/components/cover/zwave.py

```diff
@@ -81,6 +66,8 @@ def update_properties(self):
         self._close_id = self.get_value(
             class_id=zwave.const.COMMAND_CLASS_SWITCH_MULTILEVEL,
             label=['Close', 'Down'], member='value_id')
+        if self._workaround == workaround.WORKAROUND_REVERSE_OPEN_CLOSE:
```

Comment:
No, it is this specific device FGRM222.
See:
https://github.com/openhab/openhab1-addons/issues/1060
https://groups.google.com/forum/#!topic/openzwave/AD4QUaY9vQI